cfn-resources/network-peering/cmd/resource/resource.go

```diff
@@ -200,33 +186,35 @@ func Update(req handler.Request, prevModel *Model, currentModel *Model) (handler
 	}
 
 	peerID := *currentModel.Id
-	peerRequest := mongodbatlas.Peer{}
+	peerRequest := admin.BaseNetworkPeeringConnectionSettings{}
 
 	region := currentModel.AccepterRegionName
 	if region != nil {
-		peerRequest.AccepterRegionName = *region
+		peerRequest.AccepterRegionName = region
 	}
 	accountID := currentModel.AwsAccountId
```

Comment:
knitpit for all:

if accountID := currentModel.AwsAccountId; accountID != nil {
	peerRequest.AwsAccountId = accountID
}

Reply:
it makes sense. Updated